bigtable/bttest/store_leveldb_disk.go

```diff
@@ -41,7 +41,7 @@ func (f LeveldbDiskStorage) Create(tbl *btapb.Table) Rows {
 func (f LeveldbDiskStorage) GetTables() []*btapb.Table {
 	// Ignore any errors, just return
 	var ret []*btapb.Table
-	err := filepath.Walk(f.Root, func(path string, info os.FileInfo, err error) error {
+	err := filepath.Walk(f.Root, func(path string, info os.FileInfo, _ error) error {
```

Comment:
this err parameters seems not to be used by the function. not sure if we can update this file. the problem was found on staticcheck

Reply:
LG